app/views/admin/statistics_announcements/_search_results.html.erb

```diff
@@ -0,0 +1,59 @@
+<p class="govuk-heading-s govuk-!-margin-bottom-3"><%= pluralize(number_with_delimiter(paginator.total_count), "document") %></p>
+
+<% if paginator.blank? %>
+  <p class="govuk-body app-view-edition-search-results__no_documents">No <%= filter.options[:dates] %> statistics announcements found</p>
+<% else %>
+  <div class="govuk-table--with-actions">
+    <%= render "govuk_publishing_components/components/table", {
+      head: [
+        {
+          text: "Title"
+        },
+        {
+          text: "Organisation"
+        },
+        {
+          text: "Release date"
+        },
+        {
+          text: "Announcement"
+        },
+        {
+          text: tag.span("View", class: "govuk-visually-hidden")
```

Comment:
I wonder if it would make more sense for this text to be "Action" or something more descriptive of the column.

Reply:
I think I'll leave this just to be consistent with the editions search_results partial.